src/parser/pegen.ts

```diff
@@ -2118,7 +2239,7 @@ kwarg_or_starred[KeywordOrStarred*]:
     | a=starred_expression { _PyPegen_keyword_or_starred(p, a, 0) } <<-- here
     | invalid_kwarg
 */
-export function keyword_or_starred(p: Parser, element: any, is_keyword: boolean) {
+export function keyword_or_starred(p: Parser, element: keyword, is_keyword: boolean) {
```

Comment:
Maybe this is something we can inline?

Reply:
yeah probably we can do away with this whole concept since we have reflection 😄